### DISCUSSION TITLE
Oops… The properties sheet could not be loaded: Failed to fetch

### DISCUSSION THREAD

**Discussion by dirk87** [FlowmapBlue/FlowmapBlue#83] [General]
Getting this error when I try and load a new OD map:
Oops… The properties sheet could not be loaded:
Failed to fetch

Can't identify any issues in the properties sheet. Any solutions to other users having had this issue?

**Comment by ilyabo** [comment]
Resolved here: https://github.com/FlowmapBlue/FlowmapBlue/issues/84
